docs/src/modules/java-protobuf/pages/running-locally.adoc

```diff
@@ -0,0 +1,266 @@
+= Run a service locally
+:page-supergroup-java-scala: Language
+include::ROOT:partial$include.adoc[]
+
+Running a service locally is helpful to test and debug. The following sections provide commands for starting and stopping a single service locally.
+
+== Prerequisites
+
+In order to run your service locally, you'll need to have the following prerequisites:
+
+* Docker {minimum_docker_version} or higher
+* Access to the `gcr.io/kalix-public` container registry
```

Comment:
How do users know what this means, do they have access automatically, is it public, do they need to login with Kalix CLI? Clarifying that would be good.

Reply:
Good point. I don't know why we have it. This part is a copy of what we had on the main docs. 

So it's public and accessible to everyone. Users don't need to login to Kalix or whatsoever. 

Maybe the intention was to say that `gcr.io/kalix-public` should not be blocked by a firewall. 🤷